themes/ccc/components/Feature.tsx

```diff
@@ -1,44 +1,66 @@
 import Image from "next/image";
 import { ReactNode } from "react";
 
+import { Columns } from "@/components/atoms/columns/Columns";
 import { SiteWidth } from "@/components/panels/SiteWidth";
 import { Heading } from "@/components/typography/Heading";
 
+import { Card } from "./Card";
+
 interface IProps {
   heading?: string;
+  subheading?: string;
   contentSide?: "left" | "right";
   image?: string;
   imageAlt?: string;
+  inline?: boolean;
   children?: ReactNode;
 }
 
-export const Feature = ({ heading, contentSide = "left", image, imageAlt, children }: IProps) => {
+export const Feature = ({ heading, subheading, contentSide = "left", image, imageAlt, inline = false, children }: IProps) => {
+  const cardContent = (
+    <>
+      {subheading && (
+        // Because this component uses the Card component, we need to override the heading style in order to get the look we want
+        <Heading level={3} extraClasses="text-2xl !text-gray-600 !mb-6 !font-bold">
+          {subheading}
+        </Heading>
+      )}
+      {children}
+    </>
+  );
+
+  if (inline) {
+    return (
+      <Card heading={heading || ""} extraClasses="h-full" headingClasses="text-blue-900 text-2xl font-bold">
```

Comment:
We are moving away from the old blue colour set, think this is close enough to the `text-brand-darker`. See new colours in the `theme.css` file at the bottom.